include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -735,7 +735,7 @@ namespace RINGMesh
     /*!
      * @brief Build a GeoModel from a Gocad Model3D (file_model.ml)
      */
-    class RINGMESH_API GeoModelBuilderML final : public GeoModelBuilderGocad
+    class io_api GeoModelBuilderML final : public GeoModelBuilderGocad
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "public virtual" or "protected" destructor to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1235)